tests/common-unit/hierarchical-menu/index.ts

```diff
@@ -0,0 +1,10 @@
+import type { TestSetup } from '../common';
```

Comment:
I wouldn't categorize this as `common-unit`, this distinction is ambiguous IMO. Maybe just `tests/common/widgets/...`?

Reply:
tests/common/widgets makes sense. Should the name of the package also be `@instantsearch/tests/widgets` or still from `@instantsearch/tests` directly?

Reply:
I think `@instantsearch/tests` works, because we can later write common tests for other pieces (e.g., connector usage) or full apps.